library/Fission/AWS/Types.hs

```diff
@@ -1,11 +1,13 @@
 -- | Fission-speciric AWS types
```

Comment:
(sp): specific
(I know this isn't your change but just saw it now)

Reply:
haha kk will fix